homeassistant/components/media_player/webostv.py

```diff
@@ -217,7 +221,7 @@ def is_volume_muted(self):
     @property
     def volume_level(self):
         """Volume level of the media player (0..1)."""
-        return self._volume / 100.0
+        return float(self._volume) / 100.0
```

Comment:
This shouldn't be necessary. Dividing by a float will perform the conversion automatically.